examples/asr/conf/speech_multitask/fast-conformer_aed.yaml

```diff
@@ -83,7 +83,7 @@ model:
 
     custom_tokenizer:
       _target_: nemo.collections.common.tokenizers.canary_tokenizer.CanaryTokenizer  # Can be replaced with other tokenizer for different prompt formats
-      tokenizers: null  # Filled at runtime by all the tokenizers inside the aggregate tokenizer
+      tokenizer: null
```

Comment:
can you either update this config with default special tokens or update the example at the top of aed multitask training script?